contracts/vesting/src/tests/instantiate.rs

```diff
@@ -0,0 +1,259 @@
+use curve::{Curve, SaturatingLinear};
+use soroban_sdk::{
+    testutils::{Address as _, Ledger},
+    vec, Address, Env, String,
+};
+
+use crate::{
+    storage::{MinterInfo, VestingBalance, VestingTokenInfo},
+    tests::setup::instantiate_vesting_client,
+};
+
+#[test]
+fn instantiate_contract_succesffuly() {
```

Comment:
Correct this everywhere.

```suggestion
fn instantiate_contract_successfully() {
```